Fix: Improve spacing/padding consistency accross DSP Settings

maximmaxim345 opened the pull request:
Previously Input settings had considerably less padding then Output settings and the Tone Controls filter. This PR adds the missing `<v-card-item>` to fix this.
Previously padding was inconsistent across the DSP Settings. 
This PR addresses this by using `<v-card-item>` for items inside the `<v-card>`, except `<DSPSlider>`, which already has padding on its own.
Additionally this PR makes labels of `<DSPSlider>` a constant width.

# Screenshot of current beta
![image](https://github.com/user-attachments/assets/493d1d2e-73f3-4162-b74c-16641d97b0bd)

# Screenshots with this PR

![input](https://github.com/user-attachments/assets/3e4f3caa-bdf3-411f-b367-f1f25a243a33)
![tone control](https://github.com/user-attachments/assets/8e967ee2-f6a1-49a1-b258-8f935ee7dd00)
![peq](https://github.com/user-attachments/assets/9481a3ce-fc5d-4858-a2d2-ef67f9980656)
![output](https://github.com/user-attachments/assets/255371f3-57b2-4316-8cd3-de0e3a70a8d4)

marcelveldt commented:
question: why not remove the padding alltogether and remove the v-card-item from the output card ?

maximmaxim345 committed:
Fix: remove unnecessary padding in DSPToneControl

maximmaxim345 committed:
Fix: Make label text width of DSPSlider constant

100px seems to work fine in english. In case the label is larger than
100px, we will just loose symmetry.

maximmaxim345 committed:
Fix: Make padding in DSPParametricEQ look identical to all other DSP settings

maximmaxim345 committed:
Fix: Remove double padding in Output DSP settings

maximmaxim345 committed:
Fix: Hide unused details to remove whitespace

maximmaxim345 commented:
> question: why not remove the padding alltogether and remove the v-card-item from the output card ?

Good point! I changed the PR so that `<v-card-item>` (or an equivalent padding) is only used on non sliders.
I also tried to remove padding completely, but that hurt readability by a bit.

marcelveldt reviewed (approved):
Looks good!
